gateway_code/autotest/autotest.py

```diff
@@ -118,6 +118,7 @@ def _measures_handler(self, measure_str):
     @staticmethod
     def get_local_mac_addr():
         """ Get eth0 mac address """
+        return "aa:aa:aa:aa:aa:aa"
```

Comment:
not sure how to fix this

Reply:
in MAC_CMD 
eth0 => enp0s25 on my machine, but we should have something less fragile

Reply:
You could just take the name of the interface from 'config' or use an env variable.

Reply:
Thanks @cladmi, I'll look into it tomorrow.